site/layouts/shortcodes/helm-version.html

```diff
@@ -14,6 +14,9 @@
 {{- with (strings.HasPrefix $pagePrefix "v1.4") -}}
 {{- "v1.4.2" -}}
 {{- end -}}
+{{- with (strings.HasPrefix $pagePrefix "v1.5") -}}
```

Comment:
Needs to be a in a different PR

Reply:
After re-reading https://gateway.envoyproxy.io/contributions/releasing/#prerequisites-2 it also looks like I should've opened this against main